### ISSUE TITLE
How Is The Management:Endpoint:Path property used?

### ISSUE THREAD

**Issue by JackInTheBoxBen** [SteeltoeOSS/Steeltoe#1150]
## Question
I would like to redirect the path to the actuator endpoints globally. The documentation says it can be done like this?

Management": {
        "Endpoints": {
            "Path": "your url here",
            "Actuator": {
                "Exposure": {
                    "Include": [
                        "*"
                    ]
                }
            }
        }

Is this correct as anything I change it to does not work. I just get a blank page with no actuators listed. Am I setting this correctly?

## Environment (please complete the following information):
 - .NET Version 7
 - Steeltoe Version 3.2.3
## Additional context or links
I would like the url to the actuators to looks something like this:
http://www.anydomain.com/Employment/Queries/actuator or
http://www.anydomain.com/Employment/Queries/actuator/env etc

**Comment by hananiel** [comment]
Your understanding is correct. I just tested with this `appsettings.json`:

```
 "Management": {
    "Endpoints": {
      "Path": "/Employment/Queries/actuator",
      "Actuator": {
        "Exposure": {
          "Include": [ "*" ]
        }
      }
    }
  },
```
Running locally, I was able to access the endpoints at "https://[localhost:5001/Employment/Queries/actuator"

Are using PathBase middeware or hosting at a different context? Or perhaps giving the full url in the `Path` setting? Or perhaps expecting it to start with actuator when actually setting the `Path` to /Employment/Queries
